huaweicloud/services/antiddos/resource_huaweicloud_antiddos_basic.go

```diff
@@ -245,3 +278,22 @@ func getAntiDdosStatus(client *golangsdk.ServiceClient, antiddosID string) resou
 		return s, s.Status, nil
 	}
 }
+
+type antiDDoSNotFoundError struct {
+	// Error code
+	ErrCode string `json:"error_code"`
+	// Error message
+	ErrMessage string `json:"error_msg"`
+}
+
+func parseAntiDDoSQueryError(respErr error) error {
+	var apiError antiDDoSNotFoundError
+
+	if errCode, ok := respErr.(golangsdk.ErrDefault403); ok {
+		pErr := json.Unmarshal(errCode.Body, &apiError)
```

Comment:
you can try to use `common.ParseErrorMsg(errCode.Body)`

Reply:
okey, it works